src/UserHydrator.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Icepace;
+
+use PDO;
+
+class UserHydrator
+{
+    public static function getAllUsers(PDO $db): array
+    {
+        $query = $db->prepare('SELECT `id`, `username`, `hashed_pass`, `bio`, `avatar` FROM `users`');
+        $query->setFetchMode(PDO::FETCH_CLASS|PDO::FETCH_PROPS_LATE, User::class);
```

Comment:
need to have `use Icepace\User;` in order for this fetch mode to work

Reply:
change left undone